File: src/targets/javascript/jquery/fixtures/postdata-malformed.js
const settings = {
  async: true,
  crossDomain: true,
  url: 'https://httpbin.org/anything',
  method: 'POST',
  headers: {
    'content-type': 'application/json'
  }
};

$.ajax(settings).done(res => {
  console.log(res);
});

File: src/targets/ocaml/cohttp/fixtures/postdata-malformed.ml
open Cohttp_lwt_unix
open Cohttp
open Lwt

let uri = Uri.of_string "https://httpbin.org/anything" in
let headers = Header.add (Header.init ()) "content-type" "application/json" in

Client.call ~headers `POST uri
>>= fun (res, body_stream) ->
  (* Do stuff with the result *)